lab07-jacob/server.js

```diff
@@ -0,0 +1,46 @@
+const http = require('http');
+const url = require('url');
+const queryString = require('querystring');
+const cowsay = require('cowsay');
+const parseBody = require('./lib/parse-body');
+
+const PORT = process.env.PORT || 3000;
+
+const server = http.createServer();
+
+
+server.on('request', function(req, res) {
+  var path = url.parse(req.url).pathname;
+  var str = req.url.split('?')[1];
+  var query = queryString.parse(str);  //thank you http://stackoverflow.com/questions/18769673/get-querystring-in-node-js
+  if (req.method === 'POST' && path === '/cowsay') {
+    parseBody(req, function() {
+      if (req.body.text) { //if there is a text property in the body object of the request object
```

Comment:
I mentioned this in lecture, but please do create a separate branch for your comments, and then clean up the code before you submit. It's a lot to look at when reading it on screen.